rubygem/lib/zeus/m.rb

```diff
@@ -25,7 +25,7 @@ module Zeus
   #
   #[![m ci](https://secure.travis-ci.org/qrush/m.png)](http://travis-ci.org/qrush/m)
   #
-  #![Rush is a heavy metal band. Look it up on Wikipedia.](https://raw.github.com/qrush/m/master/rush.jpg)
+  #![Rush is a heavy metal band. Look it up on Wikipedia.](https://raw.github.com/qrush/m/coordinator/rush.jpg)
```

Comment:
I think the find and replace went to far here as this is not part of this repo.

Reply:
You are correct, I'll adjust.